src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  background: rgb(0,0,0);
+}
+
+div {
+  border: 1px solid #e75551;
+  box-sizing: border-box;
+
```

Comment:
you make yourself less efficient if you need to fix your homework only because you forgot to check formatting before submitting

Reply:
is it about empty space after border box?

Reply:
yup